packages/chip/src/Chip/Chip.spec.tsx

```diff
@@ -245,6 +247,22 @@ describe('packages/chip', () => {
     });
   });
 
+  describe('glyph', () => {
+    test('renders when glyph is passed', () => {
+      const { queryByTestId } = renderChip({
+        glyph: <Icon glyph="Wizard" data-testid="chip-glyph" />,
+      });
+      const icon = queryByTestId('chip-glyph');
```

Comment:
nit: `queryByTestId` is useful for asserting that an element is not present, but `getByTestId` makes more sense for this case to get a descriptive error if the expected element is not present

https://testing-library.com/docs/queries/about/#types-of-queries